.github/workflows/ci.yaml

```diff
@@ -400,8 +404,81 @@ jobs:
         env:
           POSTGRES_VERSION: "13"
           TS_DEBUG_DISCO: "true"
+        shell: bash
         run: |
-          make test-postgres
+          # if macOS, install google-chrome for scaletests
+          # As another concern, should we really have this kind of external dependency
+          # requirement on standard CI?
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            brew install google-chrome
+          fi
+
+          # By default Go will use the number of logical CPUs, which
+          # is a fine default.
+          PARALLEL_FLAG=""
+
+          # macOS will output "The default interactive shell is now zsh"
+          # intermittently in CI...
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            touch ~/.bash_profile && echo "export BASH_SILENCE_DEPRECATION_WARNING=1" >> ~/.bash_profile
+          fi
+
+          if [ "${{ runner.os }}" == "Linux" ]; then
+            make test-postgres
+          else
+            go run scripts/embedded-pg/main.go
+            DB=ci gotestsum --format standard-quiet -- -v -short -count=1 ./...
+          fi
+
+      - name: Upload test stats to Datadog
+        timeout-minutes: 1
+        continue-on-error: true
+        uses: ./.github/actions/upload-datadog
+        if: success() || failure()
+        with:
+          api-key: ${{ secrets.DATADOG_API_KEY }}
+
+  # NOTE: this could instead be defined as a matrix strategy, but we want to
+  # temporarily allow windows tests to fail. Using a matrix strategy here makes
```

Comment:
I see the Windows tests are particularly flaky, I'm not sure if we want to merge a CI suite that often fails, even if it's not required. Perhaps it should just get skipped/commented out until we're able to improve the reliability?

Reply:
Since we do have the Windows tests covered by in memory for now, maybe these should be disabled for now to avoid :x: appearing everywhere. Are there any issues tracking the windows tests breaking in postgres?

Reply:
> Are there any issues tracking the windows tests breaking in postgres?

I just added https://github.com/coder/coder/issues/15560. 

> for now, maybe these should be disabled for now to avoid ❌ appearing everywhere

Makes sense. I'll postpone merging this PR until Windows tests are fixed. I'll try to get it done this week.